modules/EmergencyMedicine/sql/0_1_0-to-0_2_0_upgrade.sql

```diff
@@ -485,3 +485,15 @@ INSERT INTO `code_types` (`ct_key`, `ct_id`, `ct_seq`, `ct_mod`, `ct_just`, `ct_
 ('BK Diseases', 9921, 9921, 12, 'BK Diseases', '', 1, 0, 1, 0, 1, 'Background Diseases', 0, 1, 0, 1, 1, 0);
 #EndIf
 
+#specialSql
+
+delimiter $$
```

Comment:
I don't think `delimiter $$` is working
I my test (in addictions procedures) need to put it without `delimiter $$` and it works just like that.

Reply:
But it runs now "as is" because of the  #specialSql tag It should work now (hopefully)
Lets upgrade and see , I do not have installation to test it on, do you have ?

Reply:
why do not you have? It's regular installation